src/common/proc_macro/src/session_config.rs

```diff
@@ -236,6 +245,13 @@ pub(crate) fn derive_config(input: DeriveInput) -> TokenStream {
                 Default::default()
             }
 
+            fn alias_to_entry_name(key_name: &str) -> String {
```

Comment:
No need `to_string` here?

```suggestion
            fn alias_to_entry_name(key_name: &str) -> &str {
```